beacon_node/network/src/beacon_processor/worker/rpc_methods.rs

```diff
@@ -246,30 +246,52 @@ impl<T: BeaconChainTypes> Worker<T> {
                                 "request_root" => ?root
                             );
                         }
-                        Ok((Some(_), None)) => {
+                        Ok((Some(block), None)) => {
+                            let finalized_data_availability_boundary = self.chain.finalized_data_availability_boundary();
+                            let block_epoch = block.epoch();
+
+                           if Some(block_epoch) >= finalized_data_availability_boundary {
+                                debug!(
+                                    self.log,
+                                    "Peer requested block and blob that should be available, but no blob found";
+                                    "peer" => %peer_id,
+                                    "request_root" => ?root,
+                                    "finalized_data_availability_boundary" => finalized_data_availability_boundary,
+                                );
+                            } else {
```

Comment:
I haven't looked into other reasons for down-scoring in the codebase, so can't say if spamming us with this message is bad enough. Down-scoring and banning peers should be kept to a minimum since mistakes happen in a p2p network, so I'll let you and @realbigsean debate this sensitive one.

Reply:
I think you are right. We should just log the incorrect request, specially in early testnets phase.